buildenv/jenkins/JenkinsfileBase

```diff
@@ -575,6 +575,10 @@ def post(output_name) {
 				failIfNoResultsValue = false
 			}
 			step([$class: "TapPublisher", testResults: "**/*.tap", outputTapToConsole: false, failIfNoResults: failIfNoResultsValue])
+			
+			if(params.UPSTREAM_TEST_JOB_NAME && params.UPSTREAM_TEST_JOB_NUMBER){
```

Comment:
- Please add comments for why we only archive TAP for child jobs. For example, only archive TAP file for child jobs. And the file will be copied into the parent job after parallel runs
- nitpicking: space before `(` and after `)`